roles/openshift_version/tasks/main.yml

```diff
@@ -86,8 +86,16 @@
   include: set_version_rpm.yml
   when: not is_containerized | bool
 
-- name: Set openshift_version for containerized installation
-  include: set_version_containerized.yml
+- block:
+  - name: Set openshift_version for containerized installation
+    include: set_version_containerized.yml
+  - name: Determine openshift rpm version
+    include: rpm_version.yml
+  - name: Fail if rpm version and docker image version are different
```

Comment:
We've been moving towards ``assert`` rather than ``fail``, but I won't block on it.